moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -185,7 +183,18 @@ const useColumns = ({
         valueFormatter: (value) =>
           !!value?.fund_name ? `${value?.fund_id} | ${value?.fund_name}` : "",
         renderEditCell: (props) => (
-          <FundAutocompleteComponent {...props} data={data.moped_funds} />
+          <LookupAutocompleteComponent
+            {...props}
+            name={"fund"}
```

Comment:
I noticed that this is the only autocomplete without the `fullWidthPopper` prop, and I can't see a reason not to add here to keep these consistent in this table. Could you update? Thanks!

I tested adding to the subproject table, and it can make the dropdown really wide with longer project full names so I think that one is ok to leave without the full width dropdown.